code/datums/jobs.dm

```diff
@@ -215,7 +215,7 @@ ABSTRACT_TYPE(/datum/job/command)
 	announce_on_join = 1
 
 
-#ifdef SUBMARINE_MAP
+#ifdef FLEET_MAP
```

Comment:
having mixed normal `ifdef` and value-associated `define`s is gross, should just choose one

Reply:
so the request changes thing, how would i allow that or is that supposed to be a "go in and fix it" thing

Reply:
you need to fix it.

Reply:
i dunno what you really mean by that, the ifdef part. thats the way it normally is and i replaced it with fleet

Reply:
as in, how would i change this to be more suitable for you?

Reply:
you gate with an ifdef but assign a value of 1 that you never check